2023-02-10 16:54:03,528 INFO    MainThread:2976 [wandb_setup.py:_flush():68] Configure stats pid to 2976
2023-02-10 16:54:03,528 INFO    MainThread:2976 [wandb_setup.py:_flush():68] Loading settings from C:\Users\beaconproduction\.config\wandb\settings
2023-02-10 16:54:03,528 INFO    MainThread:2976 [wandb_setup.py:_flush():68] Loading settings from C:\Users\beaconproduction\sites\local\python\heywire\openai\demos\wandb\settings
2023-02-10 16:54:03,529 INFO    MainThread:2976 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-02-10 16:54:03,529 INFO    MainThread:2976 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-02-10 16:54:03,553 ERROR   MainThread:2976 [wandb_setup.py:_flush():68] error
Traceback (most recent call last):
  File "C:\Users\beaconproduction\sites\local\python\heywire\openai\venv\Lib\site-packages\wandb\sdk\wandb_init.py", line 1129, in init
    wi.setup(kwargs)
  File "C:\Users\beaconproduction\sites\local\python\heywire\openai\venv\Lib\site-packages\wandb\sdk\wandb_init.py", line 274, in setup
    wandb_login._login(
  File "C:\Users\beaconproduction\sites\local\python\heywire\openai\venv\Lib\site-packages\wandb\sdk\wandb_login.py", line 298, in _login
    wlogin.prompt_api_key()
  File "C:\Users\beaconproduction\sites\local\python\heywire\openai\venv\Lib\site-packages\wandb\sdk\wandb_login.py", line 221, in prompt_api_key
    key, status = self._prompt_api_key()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\beaconproduction\sites\local\python\heywire\openai\venv\Lib\site-packages\wandb\sdk\wandb_login.py", line 201, in _prompt_api_key
    key = apikey.prompt_api_key(
          ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\beaconproduction\sites\local\python\heywire\openai\venv\Lib\site-packages\wandb\sdk\lib\apikey.py", line 134, in prompt_api_key
    key = input_callback(api_ask).strip()
          ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\beaconproduction\sites\local\python\heywire\openai\venv\Lib\site-packages\click\termui.py", line 168, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\beaconproduction\sites\local\python\heywire\openai\venv\Lib\site-packages\click\termui.py", line 150, in prompt_func
    raise Abort() from None
click.exceptions.Abort
2023-02-10 16:54:03,555 INFO    MainThread:2976 [wandb_init.py:_log_setup():492] Logging user logs to C:\Users\beaconproduction\sites\local\python\heywire\openai\demos\wandb\run-20230210_165403-a9o6v818\logs\debug.log
2023-02-10 16:54:03,555 INFO    MainThread:2976 [wandb_init.py:_log_setup():493] Logging internal logs to C:\Users\beaconproduction\sites\local\python\heywire\openai\demos\wandb\run-20230210_165403-a9o6v818\logs\debug-internal.log
2023-02-10 16:54:03,555 INFO    MainThread:2976 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002B24F8ED2D0>
2023-02-10 16:54:03,555 INFO    MainThread:2976 [wandb_init.py:init():532] calling init triggers
2023-02-10 16:54:03,556 INFO    MainThread:2976 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-02-10 16:54:03,556 INFO    MainThread:2976 [wandb_init.py:init():588] starting backend
2023-02-10 16:54:03,556 INFO    MainThread:2976 [wandb_init.py:init():592] setting up manager
2023-02-10 16:54:03,565 INFO    MainThread:2976 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-02-10 16:54:03,574 INFO    MainThread:2976 [wandb_init.py:init():599] backend started and connected
2023-02-10 16:54:03,581 INFO    MainThread:2976 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-02-10 16:54:03,581 INFO    MainThread:2976 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-02-10 16:54:03,581 INFO    MainThread:2976 [wandb_init.py:init():687] updated telemetry
2023-02-10 16:54:03,663 INFO    MainThread:2976 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-02-10 16:54:04,002 INFO    MainThread:2976 [wandb_run.py:_on_init():2134] communicating current version
2023-02-10 16:54:04,048 INFO    MainThread:2976 [wandb_run.py:_on_init():2143] got version response 
2023-02-10 16:54:04,048 INFO    MainThread:2976 [wandb_init.py:init():775] starting run threads in backend
2023-02-10 16:54:04,286 INFO    MainThread:2976 [wandb_run.py:_console_start():2114] atexit reg
2023-02-10 16:54:04,286 INFO    MainThread:2976 [wandb_run.py:_redirect():1969] redirect: 3
2023-02-10 16:54:04,286 INFO    MainThread:2976 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-02-10 16:54:04,286 INFO    MainThread:2976 [wandb_run.py:_redirect():2059] Redirects installed.
2023-02-10 16:54:04,287 INFO    MainThread:2976 [wandb_init.py:init():817] run started, returning control to user process
2023-02-10 16:54:04,288 INFO    MainThread:2976 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-02-10 16:54:04,288 INFO    MainThread:2976 [wandb_init.py:_pause_backend():403] pausing backend
2023-02-10 17:10:16,028 WARNING MsgRouterThr:2976 [router.py:message_loop():77] message_loop has been closed
